### ISSUE TITLE
[Bug] ConnectivityCheck flags parallel way as connectable

### ISSUE THREAD

**Issue by atiannicelli** [osmlab/atlas-checks#504]
**Describe the bug**
The ConnectivityCheck flags the connection points between parallel ways that connect to parallel ways that contain bridge=yes tags

**To Reproduce**
Execute the Connectivity Check on any country with dual highways that have bridges.

**Expected behavior**
Nodes that are the connecting nodes between two ways should not be considered for connectivity check. 
Here is a Map Roulette task specifying that the node at the end of a bridge should be connected to the endpoint of the parallel bridge.
https://maproulette.org/challenge/14988/task/69492326
Here is the RapiD view of it: https://mapwith.ai/rapid#background=Bing&disable_features=boundaries&id=w314513554&map=22.89/-7.67993/109.05203
These types of nodes (ones that connect a bridge to a way) should not be flagged to be connected to the parallel ways.
Probably the connectivity check needs an additional requirement that:
if node is (NOT the terminating node of a way) and (NOT a connecting node between two ways) then
the angle of the one of the ways that include that node to a possible connecting way must be greater than 30 to consider it as a flagged node. 
In other words, if two ways are parallel (or close to parallel) then they should not be flagged.

**Screenshots & Stack traces**
![Screen Shot 2021-02-09 at 4 00 26 PM](https://user-images.githubusercontent.com/5851115/107427940-efedd000-6aef-11eb-8816-f9d4d18014f7.png)
![Screen Shot 2021-02-09 at 4 00 49 PM](https://user-images.githubusercontent.com/5851115/107427974-fe3bec00-6aef-11eb-8283-56595709363e.png)
Here is another parallel street that just gets close to the other road...
![Screen Shot 2021-02-09 at 4 03 16 PM](https://user-images.githubusercontent.com/5851115/107428233-570b8480-6af0-11eb-869c-29550cf51e2f.png)

**Environment (please complete the following information):**
 - OS: [All]
 - Version [6.1.12]

**Additional context**
